### DISCUSSION TITLE
how to ignore Druid/Connection/getConnection,Druid/Connection/close and so on

### DISCUSSION THREAD

**Discussion by czxin788** [apache/skywalking#8767] [Q&A] [answered]
how to ignore Druid/Connection/getConnection,Druid/Connection/close and so on 
![image](https://user-images.githubusercontent.com/13861904/160781419-f64729c5-df43-49b4-aede-210b44939745.png)

**Comment by czxin788** [comment]
up

**Comment by czxin788** [comment]
trace.ignore_path=Druid/Connection/getConnection/**,Druid/Connection/close/**
agent.ignore_suffix=${SW_AGENT_IGNORE_SUFFIX:.jpg,.jpeg,.js,.css,.png,.bmp,.gif,.ico,.mp3,.mp4,.html,.svg,getConnection,close}

but don't work,I don't know the reaseon

**Reply by xunyao4dev** [reply]
I found that this plugin can only ignore the top level path via testing

**Comment by wu-sheng** [comment, marked as answer]
Ignore is segment oriented, aka first span in a process/thread.

You could ignore one inside the process, otherwise, the context is easy to leak, and codes re hard to write.
